messaging/app/src/main/java/com/google/firebase/quickstart/fcm/kotlin/MainActivity.kt

```diff
@@ -84,6 +88,32 @@ class MainActivity : AppCompatActivity() {
         Toast.makeText(this, "See README for setup instructions", Toast.LENGTH_SHORT).show()
     }
 
+    override fun onResume() {
+        super.onResume()
+
+        // [START send_reg_token]
+        FirebaseMessaging.getInstance().token
```

Comment:
Since this is a Kotlin snippet, we might want to demonstrate KTX here
```suggestion
        Firebase.messaging.token
```